programs/server/Server.cpp

```diff
@@ -1546,15 +1558,6 @@ try
             total_memory_tracker.setDescription("(total)");
             total_memory_tracker.setMetric(CurrentMetrics::MemoryTracking);
 
-            if (cgroups_memory_usage_observer)
```

Comment:
I am still a bit worried that we removed the hard/soft limit from the config.
But probably nobody used it anyway.